src/main/java/frc/robot/note_tracking/NoteMapElement.java

```diff
@@ -6,4 +6,4 @@
 
 import edu.wpi.first.math.geometry.Translation2d;
 
-public record NoteMapElement(double expiresAt, Translation2d noteTranslation) {}
+public record NoteMapElement(double expiresAt, Translation2d noteTranslation, int health) {}
```

Comment:
i would add a new constructor to this that allows you to instantiate this without providing the health. and then it just uses the default value.

having the number 10 hardcoded in so many different places is bad, it becomes very difficult for us to tune that value

once the new constructor is added, update each place where we set the health to `10` to just use the constructor that sets it to the default value